packages/block-library/src/list-item/edit.js

```diff
@@ -1,47 +1,210 @@
+/**
+ * External dependencies
+ */
+import { get, first } from 'lodash';
+
 /**
  * WordPress dependencies
  */
 import {
 	RichText,
 	useBlockProps,
 	useInnerBlocksProps,
+	BlockControls,
+	store as blockEditorStore,
 } from '@wordpress/block-editor';
-import { __ } from '@wordpress/i18n';
+import { isRTL, __, _x } from '@wordpress/i18n';
 import { createBlock } from '@wordpress/blocks';
+import { useDispatch, useSelect, select } from '@wordpress/data';
+import { ToolbarButton } from '@wordpress/components';
+import {
+	formatOutdent,
+	formatOutdentRTL,
+	formatIndentRTL,
+	formatIndent,
+} from '@wordpress/icons';
+
+function IdentUI( { attributes, clientId } ) {
```

Comment:
This should be `IndentUI`, shouldn't it? "Ident" is short for "identifier".

Reply:
Exactly nice catch @ZebulanStanphill  👍